WordPress/Classes/ViewRelated/Me/All Domains/View Models/AllDomainsListViewModel.swift

```diff
@@ -73,4 +75,40 @@ class AllDomainsListViewModel {
     func domain(atIndex index: Int) -> AllDomainsListItemViewModel {
         return .init(domain: domains[index])
     }
+
+    // MARK: - Creating Empty State View Models
+
+    /// The empty state to display when the user doesn't have any domains.
+    private func emptyStateViewModel() -> AllDomainsListEmptyStateViewModel {
+        return .init(
+            title: Strings.emptyStateTitle,
+            description: Strings.emptyStateDescription,
+            button: .init(title: Strings.emptyStateButtonTitle, action: {})
```

Comment:
I fear we might forget to wire this button up. Can you create an empty "Add domain" function and wire it to this button and the "+" button?

Reply:
Resolved in 3cf98ed7a112b9e7781589514fb8aeeffc0e2fbe